catalogue/aarch64-faults/README.md

```diff
@@ -31,3 +31,21 @@ in the folder `${KUT_DIR}`:
     % sh ./litmus/run.sh
 
 > **_NOTE:_** litmus7 does not support MTE and, consequently, cannot run the test LDRredF.
+
+Herd7 support for faults
```

Comment:
Can we indicate here that fault atoms are also supported in litmus7 (if not also the tools)?